Cargo.toml

```diff
@@ -55,25 +55,110 @@ members = [
 ]
 
 [workspace.dependencies]
+alphanumeric-sort = "1.5"
+ansi-str = "0.8"
+base64 = "0.22"
+bracoxide = "0.1.2"
+byteorder = "1.5"
+bytesize = "1.3"
+calamine = "0.24.0"
+chardetng = "0.1.17"
 chrono = { default-features = false, version = "0.4" }
+chrono-humanize = "0.2.3"
+chrono-tz = "0.8"
+crossbeam-channel = "0.5.8"
 crossterm = "0.27"
+csv = "1.3"
 ctrlc = "3.4"
+dialoguer = { default-features = false, version = "0.11" }
+digest = { default-features = false, version = "0.10" }
+dirs-next = "2.0"
+dtparse = "2.0"
+encoding_rs = "0.8"
 fancy-regex = "0.13"
+filesize = "0.2"
+filetime = "0.2"
+fs_extra = "1.3"
+fuzzy-matcher = "0.3"
+hamcrest2 = "0.3"
+heck = "0.5.0"
+human-date-parser = "0.1.1"
+indexmap = "2.2"
+indicatif = "0.17"
+is_executable = "1.0"
+itertools = "0.12"
+libc = "0.2"
+libproc = "0.14"
 log = "0.4"
+lru = "0.12"
+lscolors = { version = "0.17", default-features = false }
+lsp-server = "0.7.5"
+lsp-types = "0.95.0"
+mach2 = "0.4"
+md5 = { version = "0.10", package = "md-5"}
 miette = "7.2"
+mime = "0.3"
+mime_guess = "2.0"
+mockito = { version = "1.4", default-features = false }
+native-tls = "0.2"
 nix = { version = "0.27", default-features = false }
+notify-debouncer-full = { version = "0.3", default-features = false }
 nu-ansi-term = "0.50.0"
+num-format = "0.4"
+num-traits = "0.2"
+omnipath = "0.1"
 once_cell = "1.18"
+open = "5.1"
+os_pipe = "1.1"
 pathdiff = "0.2"
 percent-encoding = "2"
+print-positions = "0.6"
+procfs = "0.16.0"
+pwd = "1.3"
+quick-xml = "0.31.0"
+quickcheck = "1.0"
+quickcheck_macros = "1.0"
+rand = "0.8"
+ratatui = "0.26"
+rayon = "1.9"
 reedline = "0.30.0"
+regex = "1.9.5"
+ropey = "1.6.1"
+roxmltree = "0.19"
 rstest = { version = "0.18", default-features = false }
+rusqlite = "0.31"
+rust-embed = "8.2.0"
+same-file = "1.0"
+serde = { version = "1.0", default-features = false }
 serde_json = "1.0"
+serde_urlencoded = "0.7.1"
+serde_yaml = "0.9"
+sha2 = "0.10"
+strip-ansi-escapes = "0.2.0"
 sysinfo = "0.30"
+tabled = { version = "0.14.0", default-features = false }
 tempfile = "3.10"
+terminal_size = "0.3"
+titlecase = "2.0"
+toml = "0.8"
+trash = "3.3"
+umask = "2.1"
 unicode-segmentation = "1.11"
+unicode-width = "0.1"
+ureq = { version = "2.9", default-features = false }
+url = "2.2"
+uu_cp = "0.0.23"
+uu_mkdir = "0.0.23"
+uu_mktemp = "0.0.23"
+uu_mv = "0.0.23"
+uu_whoami = "0.0.23"
+uucore = "0.0.24"
```

Comment:
Not sure if the versions for `uucore` and the `uu_xxx` crates are supposed to match?

Reply:
yes, they're all supposed to match. Not sure how that snuck by.

Reply:
Alright, I've bumped them all to 0.24.